kafka-client/src/main/java/dev/responsive/kafka/internal/stores/ResponsiveSessionStore.java

```diff
@@ -228,4 +231,11 @@ public KeyValueIterator<Windowed<Bytes>, byte[]> fetch(final Bytes from, final B
   private long minValidEndTimestamp() {
     return observedStreamTime - params.retentionPeriod() + 1;
   }
+
+  public void restoreBatch(final Collection<ConsumerRecord<byte[], byte[]>> records) {
```

Comment:
happened to notice we weren't updating the streamTime during restoration for window or session stores either